AGENTS.md

```diff
@@ -0,0 +1,1440 @@
+# AI Agent Development Guide for Kiali
+
+This guide provides coding standards, development workflows, and common commands for AI agents and developers contributing to the Kiali project. It complements the existing [CONTRIBUTING.md](./CONTRIBUTING.md) and [STYLE_GUIDE.adoc](./STYLE_GUIDE.adoc).
```

Comment:
Maybe also the code of conduct guide.  Should this same blurb lead the other doc as well?